src/logger.cpp

```diff
@@ -48,6 +48,11 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_ENABLE_LOGGING)
 /// Allows to reproduce library use cases using the driver instead of the actual application.
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_ENABLE_LOGGING_CMD)
 
+/// Prints driver command lines into log when Find API is called.
+/// Works from any application which uses the library.
+/// Allows to reproduce library use cases using the driver instead of the actual application.
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_ENABLE_LOGGING_CMD_FIND)
```

Comment:
[Question] Let's agree on what is Find API for convolutions. Which API calls constitute it, what do you think?

Reply:
Regarding which API calls constitute it? If this env variable is enabled the driver command for find APIs( `miopenFindConvolutionBackwardDataAlgorithm`, `miopenFindConvolutionForwardAlgorithm` and `miopenFindConvolutionBackwardWeightsAlgorithm`) is logged. 

eg: 
`MIOpen(HIP): Command [LogCmdFindConvolution] ./bin/MIOpenDriver conv -n 100 xxx`

Reply:
I do not think we need dedicated `MIOPEN_ENABLE_LOGGING_CMD_FIND` (unless there is some use for it right now).

Reply:
I think it is useful since the Find call results in kernels being invoked. If there is a bug and the kernel is crashing during the Find Stage then its difficult to infer which kernel is actually causing the crash since the Driver command only gets logged when  the actual convolution is run. 

With a separate environment variable we can also detect if the framework is called find over and over and the count that we would get for the actual convolution operations would be accurate as well

Reply:
@JehandadKhan Sounds reasonable, especially the latter.